docs/removing-documents.md

```diff
@@ -15,18 +15,7 @@ Environment variables are being used to pass parameters to the rake tasks.
 
 ## Withdrawing documents
 
```

Comment:
I'd just remove this section since the title no-longer references Withdrawing and we then end up with a h1 of Removing Documents followed by a h2 of Removing Documents.

Reply:
Done